types/v1/labels.go

```diff
@@ -540,6 +540,14 @@ const (
 	//    This is replaced at runtime
 	JivaClusterIPHolder JivaAnnotations = "__CLUSTER_IP__"
 
+	// JivaCloneIPHolder is used as a placeholder for sync controller IP address
+	// which will be used as cloneIP
+	//
+	// NOTE:
+	//    This is replaced at runtime
+	JivaCloneIPHolder JivaAnnotations = "__CLONE_IP__"
```

Comment:
Why are we calling these as annotations? I guess this is an old type that we are re-using. But lets make it a point to avoid once we feel that this design/code is not good.

Reply:
No, these are coming from the `Volume` struct, just using this to pass as replica args .